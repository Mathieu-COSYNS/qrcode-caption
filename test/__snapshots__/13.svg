<svg xmlns="http://www.w3.org/2000/svg" width="100" height="120" viewBox="0 0 25 30" shape-rendering="crispEdges"><rect fill="#ffffff" width="25" height="30"/><path stroke="#000000" d="M0 0.5h7m1 0h1m2 0h1m2 0h1m1 0h1m1 0h7M0 1.5h1m5 0h1m1 0h1m1 0h2m4 0h1m1 0h1m5 0h1M0 2.5h1m1 0h3m1 0h1m1 0h2m1 0h1m1 0h1m1 0h1m2 0h1m1 0h3m1 0h1M0 3.5h1m1 0h3m1 0h1m2 0h2m1 0h2m2 0h1m1 0h1m1 0h3m1 0h1M0 4.5h1m1 0h3m1 0h1m1 0h1m5 0h1m1 0h1m1 0h1m1 0h3m1 0h1M0 5.5h1m5 0h1m2 0h1m1 0h1m1 0h1m2 0h1m1 0h1m5 0h1M0 6.5h7m1 0h1m1 0h1m1 0h1m1 0h1m1 0h1m1 0h7M12 7.5h2m2 0h1M0 8.5h1m2 0h8m3 0h1m1 0h2m2 0h1m1 0h3M1 9.5h1m5 0h1m2 0h2m1 0h3m1 0h1m1 0h5M0 10.5h1m3 0h3m2 0h1m1 0h3m1 0h2m1 0h2m1 0h1m2 0h1M0 11.5h2m1 0h1m3 0h3m1 0h1m2 0h2m3 0h1m1 0h4M0 12.5h1m3 0h1m1 0h2m4 0h1m2 0h4m5 0h1M0 13.5h2m1 0h3m1 0h3m4 0h3m3 0h1m2 0h1M0 14.5h2m4 0h10m1 0h1m2 0h5M0 15.5h1m1 0h4m2 0h1m2 0h1m3 0h1m3 0h2m1 0h1m1 0h1M0 16.5h1m1 0h9m3 0h1m1 0h5m1 0h2M8 17.5h6m2 0h1m3 0h1m2 0h1M0 18.5h7m1 0h4m4 0h1m1 0h1m1 0h2m2 0h1M0 19.5h1m5 0h1m1 0h1m1 0h1m1 0h1m1 0h3m3 0h1m2 0h2M0 20.5h1m1 0h3m1 0h1m1 0h1m1 0h1m1 0h10m1 0h2M0 21.5h1m1 0h3m1 0h1m1 0h1m1 0h1m3 0h2m1 0h3m1 0h1m1 0h2M0 22.5h1m1 0h3m1 0h1m2 0h2m1 0h1m1 0h1m1 0h2m1 0h2m1 0h3M0 23.5h1m5 0h1m2 0h1m4 0h1m3 0h3m1 0h3M0 24.5h7m1 0h3m2 0h2m1 0h1m4 0h1m2 0h1"/><text y="30" x="50%" text-anchor="middle" font-size="5">Value: P12</text></svg>