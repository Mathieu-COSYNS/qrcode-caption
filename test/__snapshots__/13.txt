data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='100' height='117.6' viewBox='0 0 25 29.4' shape-rendering='crispEdges'%3e%3crect fill='white' width='25' height='29.4'/%3e%3cpath stroke='black' d='M0 0.5h7m1 0h1m2 0h1m2 0h1m1 0h1m1 0h7M0 1.5h1m5 0h1m1 0h1m1 0h2m4 0h1m1 0h1m5 0h1M0 2.5h1m1 0h3m1 0h1m1 0h2m1 0h1m1 0h1m1 0h1m2 0h1m1 0h3m1 0h1M0 3.5h1m1 0h3m1 0h1m2 0h2m1 0h2m2 0h1m1 0h1m1 0h3m1 0h1M0 4.5h1m1 0h3m1 0h1m1 0h1m5 0h1m1 0h1m1 0h1m1 0h3m1 0h1M0 5.5h1m5 0h1m2 0h1m1 0h1m1 0h1m2 0h1m1 0h1m5 0h1M0 6.5h7m1 0h1m1 0h1m1 0h1m1 0h1m1 0h1m1 0h7M12 7.5h2m2 0h1M0 8.5h1m2 0h8m3 0h1m1 0h2m2 0h1m1 0h3M1 9.5h1m5 0h1m2 0h2m1 0h3m1 0h1m1 0h5M0 10.5h1m3 0h3m2 0h1m1 0h3m1 0h2m1 0h2m1 0h1m2 0h1M0 11.5h2m1 0h1m3 0h3m1 0h1m2 0h2m3 0h1m1 0h4M0 12.5h1m3 0h1m1 0h2m4 0h1m2 0h4m5 0h1M0 13.5h2m1 0h3m1 0h3m4 0h3m3 0h1m2 0h1M0 14.5h2m4 0h10m1 0h1m2 0h5M0 15.5h1m1 0h4m2 0h1m2 0h1m3 0h1m3 0h2m1 0h1m1 0h1M0 16.5h1m1 0h9m3 0h1m1 0h5m1 0h2M8 17.5h6m2 0h1m3 0h1m2 0h1M0 18.5h7m1 0h4m4 0h1m1 0h1m1 0h2m2 0h1M0 19.5h1m5 0h1m1 0h1m1 0h1m1 0h1m1 0h3m3 0h1m2 0h2M0 20.5h1m1 0h3m1 0h1m1 0h1m1 0h1m1 0h10m1 0h2M0 21.5h1m1 0h3m1 0h1m1 0h1m1 0h1m3 0h2m1 0h3m1 0h1m1 0h2M0 22.5h1m1 0h3m1 0h1m2 0h2m1 0h1m1 0h1m1 0h2m1 0h2m1 0h3M0 23.5h1m5 0h1m2 0h1m4 0h1m3 0h3m1 0h3M0 24.5h7m1 0h3m2 0h2m1 0h1m4 0h1m2 0h1'/%3e%3ctext y='28.3' x='50%25' text-anchor='middle' font-family=%22Verdana%2c 'Bitstream Vera Sans'%2c 'DejaVu Sans'%2c Tahoma%2c Geneva%2c Arial%2c Sans-serif%22 font-size='4' fill='black'%3eValue: P12%3c/text%3e%3c/svg%3e